OpenRA.Game/Platform.cs

```diff
@@ -94,12 +94,24 @@ static string GetSupportDir()
 			return dir + Path.DirectorySeparatorChar;
 		}
 
-		public static string GameDir { get { return AppDomain.CurrentDomain.BaseDirectory; } }
+		public static string GameDir { get { return gameDir.Value; } }
+		static Lazy<string> gameDir = Exts.Lazy(GetGameDir);
```

Comment:
`Lazy` for GetGameDir is unnecessary. GetGameDir is called always, only once anyway, is trivial property, IMO can/should be reverted.